server/log/main.go

```diff
@@ -45,7 +45,7 @@ var storageMap = make(map[int64]storage.LogStorage)
 
 // TODO(Martin2112): Needs to be able to swap out for different storage type
 func simpleMySQLStorageProvider(treeID int64) (storage.LogStorage, error) {
-	return mysql.NewLogStorage(trillian.LogID{[]byte("TODO"), treeID}, *mysqlURIFlag)
+	return mysql.NewLogStorage(treeID, *mysqlURIFlag)
```

Comment:
Yes, I thought that too - but I figured we should design/add that when we're ready.